src/ng1/directives/overflowTooltip/singleLineOverflowTooltip.directive.js

```diff
@@ -40,7 +40,7 @@ export default function singleLineOverflowTooltip(safeTimeout, $window, $resize)
             //watch for changes to window size
             $window.addEventListener('resize', updateTooltipFn);
 
-            scope.$on('$destroy', function () {
+            element.bind('$destroy', function () {
```

Comment:
Great, just one minor change. Can you change this to use `.on` as `.bind` is a deprecated function